<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diffs</class>
 <widget class="QWidget" name="diffs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1853</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="retainRadioButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>保留原有数据和新增</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="replaceRadioButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>替换全部数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leftOpenBT">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leftPathLE">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>xodr文件：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rightPathLE">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>50</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="rightOpenBT">
   <property name="geometry">
    <rect>
     <x>1420</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>50</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>xml文件：</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="leftTreeWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>851</width>
     <height>501</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>s</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>t</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>zOffset</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="leftSelectAllBT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leftClearBT">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="rightTreeWidget">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>160</y>
     <width>851</width>
     <height>501</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>s</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>t</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>zOffset</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="runBT">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>执行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightClearBT">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightSelectAllBT">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightRemoveBT">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="replaceAddRadioButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>替换和新增</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readRightBT">
   <property name="geometry">
    <rect>
     <x>1510</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>重新读取</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leftRoadidLE">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="leftFilterBT">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>筛选</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightFilterBT">
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>筛选</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rightRoadidLE">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>120</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="interfaceReplaceAddRadioButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>路口替换和新增</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
